### PR TITLE
solving

### PR DESCRIPTION
**Pull request by Poznanskyi**

### CODE REVIEW

**Review by MNDonut — CHANGES_REQUESTED** [review]
Tests failed

**Comment by Oleksl888** [comment]
@Poznanskyi 

> NOTE: Attach screenshots of all created or modified pages to pull request. It's important to attach images not links to them.

File: templates/registration/logge_out.html
Comment (Oleksl888) [inline templates/registration/logge_out.html:1]: File should be named `logged_out` to be resolved correctly

**Review by Oleksl888 — CHANGES_REQUESTED** [review]
Please update screenshots

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 🌸

**Pull request closed without merging**